Parsers/ASimDns/Parsers/ASimDnsMicrosoftNXlog.yaml

```diff
@@ -1,12 +1,12 @@
 Parser:
```

Comment:
can you please attach output of the tester functions with this PR? For Schema and Data (Validate the output schema and Validate the output values)
https://learn.microsoft.com/EN-US/AZURE/sentinel/normalization-develop-parsers#validate-the-output-schema

Reply:
In my current sprint I was denied to work on this topic, so I'm only scanvenging some left over time for this, because I want to make progress with this. Given that my last attempts to save the parser functions ran into difficulties, and making it work would involve troubleshooting, I can't promise to be able to come back with this anytime soon (during the next week.) But I will try it anyway.

Reply:
@jszigetvari-nxlog Thankyou for executing the ASIM tests. Please attach few sample logs with this PR. Please follow: https://learn.microsoft.com/en-us/azure/sentinel/normalization-develop-parsers#samples-submission-guidelines

Reply:
done